rules/windows/process_creation/proc_creation_win_msiexec_dll.yml

```diff
@@ -5,16 +5,20 @@ description: Detects MsiExec loading a DLL and calling its DllUnregisterServer f
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1218.007/T1218.007.md
     - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec
+    - https://twitter.com/_st0pp3r_/status/1583914515996897281
 author: frack113
 date: 2022/04/24
+modified: 2022/10/23
 logsource:
     product: windows
     category: process_creation
 detection:
     selection:
         Image|endswith: '\msiexec.exe'
-        CommandLine|contains|all:
+        CommandLine|contains:
             - ' /z '
+            - ' -z '
+        CommandLine|contains|all:
```

Comment:
You will no pass the test and get a `has the selection (selection/CommandLine|contains|all) with a list of only 1 value in detection`
You have to the 2 `CommandLine|contains` in 2 selection like this
```yaml
    selection_msi:
        Image|endswith: '\msiexec.exe'
        CommandLine|contains:
            - ' /z '
            - ' -z '
    selection_dll:
        CommandLine|contains: '.dll'
    condition: all of selection_*
```
`all of selection_*` will be translate to `selection_msi AND selection_dll`